pkgs/by-name/ha/hare/package.nix

```diff
@@ -0,0 +1,117 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, binutils-unwrapped
+, harec
+, makeWrapper
+, qbe
+, scdoc
+, tzdata
+, substituteAll
+}:
+
+let
+  # We use harec's override of qbe until 1.2 is released, but the `qbe` argument
+  # is kept to avoid breakage.
+  qbe = harec.qbeUnstable;
+  # Until #265124 is merged (it's already on staging), this override is needed.
+  tzdata' = tzdata.overrideAttrs (old: {
+    postInstall = old.postInstall + ''
+      cp leap-seconds.list $out/share/zoneinfo
+    '';
+  });
```

Comment:
```suggestion
```
lets wait until staging-next is merged https://nixpk.gs/pr-tracker.html?pr=265124